plugins/services/src/js/utils/ServiceTableUtil.js

```diff
@@ -64,6 +64,17 @@ function cpusCompareFunction(a, b) {
   return numberCompareFunction(a.getResources().cpus, b.getResources().cpus);
 }
 
+/**
+ * Compare service gpus
+ * @param {Service|ServiceTree} a
```

Comment:
Fun fact, we had an eslint rule in one of my last companies for having a certain minimum length of variables (3 it was). This resulted in comparison variables being named `aaa` or `bbb` :trollface: